azure-pipelines/builds/ci.yml

```diff
@@ -17,68 +17,6 @@ stages:
 - stage: build
   displayName: Build
   jobs:
-  - template: /eng/common/templates/jobs/jobs.yml
+  - template: /eng/common/templates/job/build-from-source.yml
     parameters:
-      enablePublishUsingPipelines: true
-      enablePublishBuildArtifacts: true
-      enablePublishBuildAssets: true
-      artifacts:
-        publish:
-          artifacts: true
-          manifests: true
-      jobs:
-      - job: sourcebuild_linux_x64
-        workspace:
-          clean: all
-        container: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-32-20200512010618-163ed2a
-        variables:
-        - name: _BuildConfig
-          value: Release
-        - name: _InternalBuildArgs
-          value: ''
-        - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-          - name: _InternalBuildArgs
-            value: >-
-              /p:DotNetPublishUsingPipelines=true
-              /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-        steps:
-
-        - script: |
-            set -x
-            df -h
-            ./build.sh --ci \
-              --configuration $(_BuildConfig) \
-              --restore --build --pack --publish \
-              $(_InternalBuildArgs) \
-              /p:ArcadeBuildFromSource=true
-          displayName: Build
-
-        # Upload build logs for diagnosis.
-        - task: CopyFiles@2
-          displayName: Prepare BuildLogs staging directory
-          inputs:
-            SourceFolder: '$(Build.SourcesDirectory)'
-            Contents: |
-              **/*.log
-              **/*.binlog
-            TargetFolder: '$(Build.StagingDirectory)/BuildLogs'
-            CleanTargetFolder: true
-          continueOnError: true
-          condition: succeededOrFailed()
-
-        - task: PublishPipelineArtifact@1
-          displayName: Publish BuildLogs
-          inputs:
-            targetPath: '$(Build.StagingDirectory)/BuildLogs'
-            artifactName: BuildLogs
-          continueOnError: true
-          condition: succeededOrFailed()
-
-# Based on - https://github.com/dotnet/arcade/blob/9b3f304c7bc9fd4d11be9ca0b466b83e98d2a191/Documentation/CorePackages/Publishing.md#moving-away-from-the-legacy-pushtoblobfeed-task
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - template: /eng/common/templates/post-build/post-build.yml
-    parameters:
-      enableSourceLinkValidation: true
-      #Nuget and Signing Validation are not needed as we only produce a transport package.
-      enableSigningValidation: false
-      enableNugetValidation: false
```

Comment:
Where'd this go?

Reply:
That will go back at the end of ci.yml.  I just didn't want it to run while I was testing.

Reply:
Don't worry about that--it'll only publish if you have a default channel assignment. (Actually, useful to validate that this is still the case. 😄)